src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3454,7 +3440,7 @@ private void GetSlotsToMarkAsNotNullable(BoundExpression operand, ArrayBuilder<i
                             if (slot > 0)
                             {
                                 // We need to continue the walk regardless of whether the receiver should be updated.
-                                var receiverType = conditional.Receiver.Type;
+                                var receiverType = conditional.Receiver.Type!;
```

Comment:
Should be assertion

Reply:
Will leave as-is since we're de-referencing immediately.

---
In reply to: [463272615](https://github.com/dotnet/roslyn/pull/45640#discussion_r463272615) [](ancestors = 463272615)